src/components/basket/BasketOrderSent.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function BasketOrderSent(props) {
+    return (
+        <div className={"basket-order__sent " + (props.orderSent === true ? '' : 'hidden')}>
```

Comment:
Ternary could be slightly shorter using `(props.orderSent ? '' : 'hidden')`, since it will already check if the value is true